packages/server/src/services/setup.ts

```diff
@@ -59,7 +60,8 @@ async function initializeAsync() {
 
     const globalDescription = globalData as ServerWorldDescription;
     const specificDescription =
-      worldSpecificData as Partial<ServerWorldDescription>;
+      fireWorldSpecificData as Partial<ServerWorldDescription>;
```

Comment:
Still seeing the issue here, feel free to comment that I'm wrong, but shouldn't this be worldSpecificData?